pyglotaran_examples/ex_spectral_constraints/models/scheme_equal_area_penalties.yml

```diff
@@ -15,7 +15,7 @@ experiments:
           - type: gaussian
             compartments:
               s1: 0.5
```

Comment:
Please also change this back to `inputs.s1`

Even when a parameter is fixed, unless it's absolutely fixed fixed I would prefer to represent it as a parameter rather than a 'magic number'.